itl/plugins-contrib.d/systemd.conf

```diff
@@ -1,51 +1,99 @@
+			description = "Exclude a systemd unit from the checks. This option can be applied multiple times."
+			repeat_key = true
 		}
-		"--warning" = {
-			value = "$systemd_warning$"
-			description = "Startup time in seconds to result in a warning status. (Default: `60s`)"
+		"--exclude-type" = {
+			value = "$systemd_exclude_type$"
+			description = "Exclude a systemd unit type (for example: `service`, `timer`)"
 		}
-		"--critical" = {
-			value = "$systemd_critical$"
-			description = "Startup time in seconds to result in a critical status. (Default: `120s`)"
+		"--state" = {
+			value = "$systemd_state$"
+			description = "Specify the active state that the systemd unit must have (for example: `active`, `inactive`)"
 		}
+
+		/* Timers related options */
 		"--dead-timers" = {
-			set_if = "$systemd_dead_timers$"
-			description = "Detect dead / inactive timers. (Default: `false`)"
+			value = "$systemd_dead_timers$"
+			description = "Detect dead / inactive timers, see `systemd_dead_timers_{warning,critical}`. (Default `false`)"
 		}
-		"--dead-timers-warning" = {
+		"--timers-warning" = {
 			value = "$systemd_dead_timers_warning$"
-			description = "Time ago in seconds for dead / inactive timers to trigger a warning state (by default 6 days)."
+			description = "Time ago in seconds for dead / inactive timers to trigger a warning state. (Default 6 days)"
 		}
-		"--dead-timers-critical" = {
+		"--timers-critical" = {
 			value = "$systemd_dead_timers_critical$"
-			description = "Time ago in seconds for dead / inactive timers to trigger a critical state (by default 7 days)."
+			description = "Time ago in seconds for dead / inactive timers to trigger a critical state. (Default 7 days)"
 		}
-		"-v" = {
-			set_if = {{ macro("$systemd_verbose_level$") == 1 }}
-			description = "Increase verbosity level (Accepted values: `1`, `2` or `3`). Defaults to none."
+
+		/* Startup time related options */
+		"--no-startup-time" = {
+			value = "$systemd_no_startup_time$"
+			description = "Don't check the startup time. Using this option, the options `systemd_{warning,critical}` have no effect. (Default `false`)"
 		}
-		"-vv" = {
-			set_if = {{ macro("$systemd_verbose_level$") == 2 }}
+		"--warning" = {
+			value = "$systemd_warning$"
+			description = "Startup time in seconds to result in a warning status. (Default 60 seconds)"
 		}
-		"-vvv" = {
-			set_if = {{ macro("$systemd_verbose_level$") == 3 }}
+		"--critical" = {
+			value = "$systemd_critical$"
+			description = "Startup time in seconds to result in a critical status. (Default 120 seconds)"
+		}
+
+		/* Monitoring data acquisition */
+		"--dbus" = {
+			value = "$systemd_dbus$"
+			description = "Use systemd's D-Bus API instead of parsing command output. Only partially implemented!"
+		}
+		"--cli" = {
+			value = "$systemd_cli$"
+			description = "Use text output from parsing command output. (Default)"
+		}
+		"--user" = {
+			value = "$systemd_user$"
```

Comment:
All of these three arguments are boolean flags, so please use `set_if` as well instead of `value`.

Reply:
Thanks, I have taken a look at all parameters again and fixed some `value`s to `set_if`.